opentelemetry-datadog/src/exporter/model/v05.rs

```diff
@@ -161,7 +161,10 @@ where
                 rmp::encode::write_u32(&mut encoded, interner.intern(key.as_str()))?;
                 rmp::encode::write_u32(&mut encoded, interner.intern(value.as_str().as_ref()))?;
             }
-            rmp::encode::write_map_len(&mut encoded, 0)?;
+            rmp::encode::write_map_len(&mut encoded, 1)?;
+            // https://github.com/DataDog/dd-trace-js/blob/c89a35f7d27beb4a60165409376e170eacb194c5/packages/dd-trace/src/constants.js#L4
+            rmp::encode::write_u32(&mut encoded, interner.intern("_sampling_priority_v1"))?;
+            rmp::encode::write_f64(&mut encoded, if span.span_context.is_sampled() { 1.0 } else { 0.0 })?;
```

Comment:
Could we define some constant at `model/mod.rs` and reuse them across v3 and v5 exporter?